README.md

```diff
@@ -12,19 +14,89 @@ then
 yarn add @mdx-js/mdx @mdx-js/loader loader-utils --dev
 ```
 
-In your gatsby-config.js
+### Configuration
+
+Add in the config for mdx and source in the `pages/` directory.
+
+`gatsby-config.js`
+
+```js
+module.exports = {
+  plugins: [
+    "gatsby-plugin-mdx",
+    {
+      resolve: "gatsby-source-filesystem",
+      options: {
+        name: "pages",
+        path: `${__dirname}/src/pages/`
+      }
+    }
+  ]
+};
 ```
-  plugins: ['gatsby-plugin-mdx'],
 
+Query all markdown files and create pages for each of them.
+
+`gatsby-node.js`
+
+```js
+exports.createPages = ({ graphql, boundActionCreators }) => {
+  const { createPage } = boundActionCreators;
+
+  return new Promise((resolve, reject) => {
+    graphql(`
+      {
+        allFile(filter: { extension: { eq: "md" } }) {
+          edges {
+            node {
+              absolutePath
+              relativeDirectory
+              name
+            }
+          }
+        }
+      }
+    `)
+      .then(result => {
+        if (result.errors) {
+          return reject(result.errors);
+        }
+
+        // Create markdown pages.
+        result.data.allFile.edges.forEach(
+          ({ node: { absolutePath, relativeDirectory, name } }) => {
+            createPage({
+              path: `/${relativeDirectory}/${name}`,
```

Comment:
I think it should be: "${relativeDirectory}/${name}" without the "/"

Reply:
Trying this out in my own site, I agree.